### DISCUSSION TITLE
Do GLEDOPTO controllers support the "Always OFF" mode after a power outage?

### DISCUSSION THREAD

**Discussion by vukitoso** [Koenkk/zigbee2mqtt#19758] [Devices]
Hello.
Please tell me, is it possible to make the lights always OFF on the GLEDOPTO GL-C-002P, GL-C-008P, GL-C-008P (mini) controllers?
The controller will always be in mode: RGB+CCT: GL-C-008P
Is it possible to configure the controller via z2m so that after turning off and on the electricity, the light is always OFF.
Example:
- I turned on the light
- the electricity in the apartment was turned off
- I went to bed
- they provide electricity at night! This is where it needs to be always OFF.

GL-C-001P was suggested to me and it definitely supports it.

**Comment by kevincaradant** [comment]
@vukitoso , have you received your answer since last year? Is the (power outage behavior)  GL-C-008P working for you at the moment, if you have it? Thanks!

**Reply by vukitoso** [reply]
> Is the (power outage behavior) GL-C-008P working for you at the moment, if you have it? Thanks!

I have GL-C-001P.
When switching to all modes in zigbee2mqtt, the power_on_behavior option works correctly.
```
[White] RGB+CCT: GL-C-008P - z2m power_on_behavior +
[Yellow] RGBW: GL-C-007P - z2m power_on_behavior +
[Blue] RGB: GL-C-003P - z2m power_on_behavior +
[Green] CCT: GL-C-006P - z2m power_on_behavior +
[Red] Dimmer: GL-C-009P - z2m power_on_behavior +
```

I make the assumption that on the GL-C-008P device in zigbee2mqtt the power_on_behavior option should also work.
Same as GL-C-002P mini.
If you buy the GL-C-008P version, please post here about the results of the power_on_behavior option.
